app/api-v1/controllers/Attachment/index.js

```diff
@@ -1,5 +1,72 @@
+const { parseAccept } = require('../../../utils/parseAcceptHeader')
+const logger = require('../../../utils/Logger')
+
+const db = require('../../../db')
+
+const { createAttachmentFromFile, createAttachment } = require('../Attachment/helpers')
+const { InternalError, NotFoundError, BadRequestError } = require('../../../utils/errors')
+
 module.exports = {
-  getAll: async function () {
-    return { status: 500, response: { message: 'Not Implemented' } }
+  get: async (req) => {
```

Comment:
The logic here has changed substantially, any reason? `NotAcceptableError` also isn't being thrown anywhere so this is clearly out of spec and a regression

Reply:
It changed as I was working with Paul, I can copy the old stuff in if you want? At the moment I'm between two people's way of doing it.

Reply:
Internal logic can change. Spec can't. I'm concerned we're re-writing something for the sake of it.